charter.md

```diff
@@ -38,6 +38,8 @@ The privacy and security considerations related to the impact of confidential co
 
 Privacy and security considerations regarding redaction, linkability and selective disclosure will be developed for proposed standards in the program of work that offer data minimization capabilities.
 
+SPICE will develop designs that follow the conceptual security model that is used in W3C VC, ISO mDocs, and similar. SPICE will not seek to extend or integrate any of those specific technologies.
```

Comment:
Minor but based on 119 meeting chatter this may be a useful suggestion, but take it or leave it, not a big deal to me.

```suggestion
SPICE will develop designs that follow the conceptual security model that is used in W3C VC, ISO mDocs, InfoCards, and similar. SPICE will not seek to extend or integrate any of those specific technologies.
```

Reply:
Is Infocards specific enough without a qualifier?  I don't know Infocards.  (I'm probably just not old enough.)

Reply:
I was in the same boat I googled and this seem to refer to a specific technology. Microsoft articles talk about it with this capitalization and Wikipedia talks about it as i-cards. People in meeting Zulip chat used the former but maybe we just go without?

I approved either way and I am not adamant in including or excluding it.